app/src/test/java/com/github/se/signify/model/user/UserRepositoryFireStoreTest.kt

```diff
@@ -607,4 +616,150 @@ class ToDosRepositoryFireStoreTest {
     // Verify that the friendUser update was attempted and failed
     verify(mockFriendUserDocRef).update(eq("friends"), ArgumentMatchers.any(FieldValue::class.java))
   }
+
```

Comment:
Test to check if the userRepo acctually changed when adding and removing.
Also tested the cases where it is supposed to fail.
Again nothing complicated simple and efficient. 
Good job